source/dotnet/wholesale-api/CalculationResults/CalculationResults.IntegrationTests/Infrastructure/CalculationResults/WholesaleResultQueriesTests.cs

```diff
@@ -0,0 +1,107 @@
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System.Globalization;
+using Energinet.DataHub.Core.Databricks.SqlStatementExecution.Internal;
+using Energinet.DataHub.Core.TestCommon.AutoFixture.Attributes;
+using Energinet.DataHub.Wholesale.Batches.Interfaces;
+using Energinet.DataHub.Wholesale.Batches.Interfaces.Models;
+using Energinet.DataHub.Wholesale.CalculationResults.Infrastructure.CalculationResults;
+using Energinet.DataHub.Wholesale.CalculationResults.Infrastructure.SqlStatements.DeltaTableConstants;
+using Energinet.DataHub.Wholesale.CalculationResults.IntegrationTests.Fixtures;
+using Energinet.DataHub.Wholesale.CalculationResults.Interfaces.CalculationResults.Model.WholesaleResults;
+using Energinet.DataHub.Wholesale.Common.Databricks.Options;
+using FluentAssertions;
+using FluentAssertions.Execution;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Moq;
+using Xunit;
+
+namespace Energinet.DataHub.Wholesale.CalculationResults.IntegrationTests.Infrastructure.CalculationResults;
+
+public class WholesaleResultQueriesTests : IClassFixture<DatabricksSqlStatementApiFixture>
+{
+    private const string CalculationId = "019703e7-98ee-45c1-b343-0cbf185a47d9";
+    private const string HourlyTariffCalculationResultId = "12345678-98ee-45c1-b343-0cbf185a47d9";
+    private const string MonthlyAmountTariffCalculationResultId = "23456789-98ee-45c1-b343-0cbf185a47d9";
+    private const string DefaultHourlyAmount = "2.34567";
+    private const string DefaultMonthlyAmount = "1.123456";
+
+    private readonly DatabricksSqlStatementApiFixture _fixture;
+
+    public WholesaleResultQueriesTests(DatabricksSqlStatementApiFixture fixture)
+    {
+        _fixture = fixture;
+    }
+
+    [Theory]
+    [InlineAutoMoqData]
+    public async Task GetAsync_ReturnsExpectedWholesaleResult(
+        Mock<IHttpClientFactory> httpClientFactoryMock,
+        Mock<ILogger<SqlStatusResponseParser>> sqlStatusResponseParserLoggerMock,
+        Mock<ILogger<DatabricksSqlStatementClient>> databricksSqlStatementClientLoggerMock,
+        Mock<IBatchesClient> batchesClientMock,
+        BatchDto batch,
+        Mock<ILogger<WholesaleResultQueries>> wholesaleResultQueriesLoggerMock)
+    {
+        // Arrange
+        var deltaTableOptions = _fixture.DatabricksSchemaManager.DeltaTableOptions;
+        await InsertHourlyTariffAndMonthlyAmountTariffRows(deltaTableOptions);
+        batch = batch with { BatchId = Guid.Parse(CalculationId) };
+        var sqlStatementClient = _fixture.CreateSqlStatementClient(
+            httpClientFactoryMock,
+            sqlStatusResponseParserLoggerMock,
+            databricksSqlStatementClientLoggerMock);
+        batchesClientMock.Setup(b => b.GetAsync(It.IsAny<Guid>())).ReturnsAsync(batch);
```

Comment:
Would it be easier to read our mock setup if we always split the lines?

```suggestion
        batchesClientMock
            .Setup(b => b.GetAsync(It.IsAny<Guid>()))
            .ReturnsAsync(batch);
```

Reply:
Maybe. I can do it :-)